components/google_analytics/actions/list-goals/list-goals.mjs

```diff
@@ -0,0 +1,70 @@
+import analytics from "../../google_analytics.app.mjs";
+
+export default {
+  key: "google_analytics-list-goals",
+  name: "List Goals",
+  description: "Lists goals to which the user has access",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    analytics,
+    token: {
+      propDefinition: [
+        analytics,
+        "token",
+      ],
+    },
```

Comment:
I don't think `token` is necessary since we have `this.$auth.oauth_access_token`.
```suggestion
```